packages/react-mutation-mapper/src/component/track/PtmTrack.tsx

```diff
@@ -26,12 +26,14 @@ type PtmTrackProps = TrackProps & {
     subTrackMargin?: number;
 };
 
-export function ptmTooltip(
-    ptms: PostTranslationalModification[],
-    pubMedCache?: MobxCache
-) {
-    return <PtmAnnotationTable data={ptms} pubMedCache={pubMedCache} />;
```

Comment:
@adamabeshouse could you comment on this? If a function returns the component, then the component will throw error in the construction method. The error is `TypeError: Cannot set property 'props' of undefined`